### PR TITLE
Ll 6594/testing

### PR DESCRIPTION
**Pull request by klambert-ledger**
Add 100% unit testing coverage to SDK.

## CONTEXT
[LL-6594](https://ledgerhq.atlassian.net/browse/LL-6594)

### CODE REVIEW

**Comment by klambert-ledger** [comment]
@IAmMorrow @achabrolin-ledger We now have to decide what % of code coverage is required (actually 100%) so I can update the `yarn test` to fail if the % is under that 👌

File: tests/unit/serializers.spec.ts [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+    });
+  });
+
+  describe("deserializeAccount", () => {
+    it("should succeed to deserialize an account", () => {
+      const deserializedAccount = serializers.deserializeAccount({
+        id: "id",
+        name: "name",
+        address: "address",
+        currency: "currentcy",
```
[the remark this answers is not in the document]
Comment (klambert-ledger) [reply tests/unit/serializers.spec.ts:48]: Good catch, will fix it ! 👍

File: tests/unit/serializers.spec.ts [hunk 2]
```diff
@@ -0,0 +1,341 @@
...
+        balance: "0",
+        spendableBalance: "0",
+        blockHeight: 0,
+        lastSyncDate: date.toString(),
+      });
+
```
[the remark this answers is not in the document]
Comment (klambert-ledger) [reply tests/unit/serializers.spec.ts:60]: Tests added for the pipe `serialize ⇢ JSON.stringify ⇢ JSON.parse ⇢ deserialize` for `Account`, `Transaction` & `SignedTransaction` 👍

**Review by achabrolin-ledger — APPROVED** [review]
Looks good to me 🚀 
Regarding % of test coverage needed, we can keep 100% as of today and adjust as we go if need be

**Comment by klambert-ledger** [comment]
> Looks good to me 🚀
> Regarding % of test coverage needed, we can keep 100% as of today and adjust as we go if need be

👍 
Squashed your typo fixes to avoid a "fix blablabla" commit, hope you don't mind !

**Review by achabrolin-ledger — APPROVED** [review]

**Review by IAmMorrow — APPROVED** [review]

**Pull request merged**